hosting_docs/source/backups_dr/4-dr-setup.rst

```diff
@@ -1,4 +1,79 @@
-Disaster Recovery Setup
```

Comment:
Just something small I noticed, not needed to rectify if you don't want to: You removed the "Setup" part from "Disaster Recovery Setup", but it's still in the name of the file :smile:

Reply:
Oh, yeah. I guess this is okay. I removed from the title since the doc talks a bit more than DR setup.